app/javascript/src/components/ViewAllCallsToggle.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from "react";
+import mount from "../mount";
+
+const ViewAllCallsToggle = ({ oldCallsCount }) => {
+  console.log(`oldCallsCount: ${oldCallsCount}`);
+  if (oldCallsCount <= 0) return null;
+
+  const [showOldCalls, setShowOldCalls] = useState(false)
+
+  const toggleCallLog = () => {
+    // in a perfect world, we'd move .old-calls into React but this is fine for now
+    oldCallsElements = document.querySelectorAll(".old-calls")
+    oldCallsElements.forEach(el => el.classList.toggle("d-none"));
+    setShowOldCalls(!showOldCalls)
+  }
+  
+  return (
+    <button className="btn btn-link" onClick={toggleCallLog}>
+      { showOldCalls ? "Limit list" : "View all calls" }
```

Comment:
is it possible to respect i18n here, or is that a bridge too far? https://github.com/i18next/react-i18next/issues/909 seems to hint that there's something in view_component that will let you get away with this. (though in this case you have found one of the few spots in the app that didn't get translated ha, so that's our mistake)

Reply:
Def not a bridge too far, I went ahead and pushed up some commits